src/components/transactionsV2/walletTransactionsV2/walletHeader.js

```diff
@@ -4,65 +4,109 @@ import { Link } from 'react-router-dom';
-            {
-              isMyWallet && <span className={`${styles.label} my-account`}>
-                {props.t('My Account')}
-              </span>
-            }
+    this.toggleRequestDropdown = this.toggleRequestDropdown.bind(this);
+    this.handleClickOutsideRequest = this.handleClickOutsideRequest.bind(this);
+    this.setRequestDropdownRef = this.setRequestDropdownRef.bind(this);
+  }
+
+  toggleRequestDropdown() {
+    if (!this.state.showRequestDropdown) {
+      document.addEventListener('click', this.handleClickOutsideRequest);
+    } else {
+      document.removeEventListener('click', this.handleClickOutsideRequest);
+    }
+
+    this.setState(prevState => ({ showRequestDropdown: !prevState.showRequestDropdown }));
+  }
+
+  // istanbul ignore next
+  handleClickOutsideRequest(e) {
+    if (this.requestDropdownRef && this.requestDropdownRef.contains(e.target)) return;
+    this.toggleRequestDropdown();
+  }
+
+  setRequestDropdownRef(node) {
+    this.requestDropdownRef = node;
+  }
+
+  render() {
+    const {
+      account, followedAccounts, address, t, match,
+    } = this.props;
+    const index = getIndexOfFollowedAccount(
+      followedAccounts,
+      { address },
+    );
+    const accountTitle = followedAccounts[index]
+      && followedAccounts[index].title;
+    const hasTitle = index !== -1 && accountTitle !== address;
+
+    const isMyWallet = match.url === routes.wallet.path;
+
+    return (
+      <header className={`${styles.wrapper}`}>
+        <div className={`${styles.account}`}>
+          <AccountVisual
+            address={account.address}
+            size={48}
+            />
+          <div className={styles.accountInfo}>
+            <div>
+              <h2 className={`${styles.title}`}>
+              { hasTitle
+                ? <span className={'account-title'}>{accountTitle}</span>
+                : <span>{t('Wallet')}</span>
+              }
+              </h2>
+              {
+                isMyWallet && <span className={`${styles.label} my-account`}>
+                  {t('My Account')}
+                </span>
+              }
+            </div>
+            <span className={styles.address}>
+              {address}
+            </span>
           </div>
-          <span className={styles.address}>
-            {props.address}
-          </span>
         </div>
-      </div>
 
-      { isMyWallet &&
-        (
-          <div className={`${styles.buttonsHolder}`}>
-            <Link to={`${routes.request.path}`} className={'help-onboarding tx-receive-bt'}>
-              <SecondaryButtonV2>
-                {props.t('Request LSK')}
-              </SecondaryButtonV2>
-            </Link>
-            <Link to={`${routes.send.path}?wallet`} className={'tx-send-bt'}>
-              <PrimaryButtonV2>
-                {props.t('Send LSK')}
-              </PrimaryButtonV2>
-            </Link>
-          </div>
-        )
-      }
-    </header>
-  );
-};
+        { isMyWallet &&
```

Comment:
@renatomassao based on what Vit say the last time I dont know if we can continue using this for show JSX code, I think you should used a ternary condition

Reply:
Done